python/tidy/servo_tidy/tidy.py

```diff
@@ -827,6 +830,17 @@ def check_config_file(config_file, print_text=True):
     if print_text:
         print '\rChecking the config file...'
 
+    config_content = toml.loads(conf_file)
+    exclude = config_content.get("ignore", {})
+
+    # Check for invalid listed ignored directories
+    exclude_dirs = exclude.get("directories", [])
+    skip_dirs = ["./target", "./tests"]
```

Comment:
We can check for `"target"` but we can safely remove the dirs (from the config) that don't exist in `"tests"`. It's old code :smile: